js/views/settingsVw.js

```diff
@@ -583,7 +600,10 @@ module.exports = Backbone.View.extend({
 
       self.setCurrentBitCoin(cCode);
       self.refreshView();
-    }, '','','','',e);
+    }).fail(() => {
+      $(e.target).removeClass('loading');
+      self.scrollToFirstError(self.$('#generalForm'));
```

Comment:
Currently, saving scrolls most tabs to some random point if there is no error (which is how it was before these changes). Could it scroll the tab to the top if everything validates instead?